src/pages/patientView/structuralVariant/StructuralVariantTableWrapper.tsx

```diff
@@ -534,16 +541,15 @@ export default class StructuralVariantTableWrapper extends React.Component<
     public render() {
         return this.tableUI.component;
     }
+}
 
-    // @action.bound
-    // private handleOncoKbIconModeToggle(mergeIcons: boolean) {
-    //     this.mergeOncoKbIcons = mergeIcons;
-    //     updateOncoKbIconStyle({ mergeIcons });
-    //
-    //     // we need to set the OncoKB width on the next render cycle, otherwise it is not updated yet
-    //     calculateOncoKbContentWidthOnNextFrame(
-    //         ANNOTATION_ELEMENT_ID,
-    //         width => (this.oncokbWidth = width || DEFAULT_ONCOKB_CONTENT_WIDTH)
-    //     );
-    // }
+function createStructVarNamespaceColumns(
+    config?: NamespaceColumnConfig
```

Comment:
is this param really optional?

Reply:
yes, when not set an empty object will be returned